src/VisualStudio/Core/Def/Implementation/Library/FindResults/TreeItems/MetadataDefinitionTreeItem.cs

```diff
@@ -40,7 +40,7 @@ public override int GoToSource()
             if (symbol != null && referencingProject != null)
             {
                 var navigationService = _workspace.Services.GetService<ISymbolNavigationService>();
-                return navigationService.TryNavigateToSymbol(symbol, referencingProject, CancellationToken.None)
+                return navigationService.TryNavigateToSymbol(symbol, referencingProject, cancellationToken: CancellationToken.None)
```

Comment:
Right. Sorry, I expressed that later on in the feedback. Yes, we ultimately hand it off to VS APIs, which we can't do anything about.